frontend/src/views/ContentView.svelte

```diff
@@ -0,0 +1,99 @@
+<style>
+  .content {
+    position: sticky;
+    top: 0;
+    height: 100%;
+    width: 100%;
+    overflow: hidden;
+  }
+
+  .breadcrumb {
+    padding-bottom: 1em;
+    background: var(--main-bg-color);
+  }
+</style>
+
+<script>
+  import { selectedResource } from "../stores";
+  import AssemblyView from "./AssemblyView.svelte";
+  import DecompilationView from "./DecompilationView.svelte";
+  import Breadcrumb from "../utils/Breadcrumb.svelte";
+  import HexView from "../hex/HexView.svelte";
+  import TextView from "./TextView.svelte";
+  import Tabs from "../utils/Tabs.svelte";
+  export let resources;
+
+  let hasTextView = false;
+  let hasAsmView = false;
+  let hasDecompView = false;
+  let tabs = [];
+  let tabId = "hex";
+
+  const hexTab = {
+    id: "hex",
+    title: "Hex",
+    component: HexView,
+    props: {
+      resources: resources,
+    },
+  };
+
+  const textTab = {
+    id: "text",
+    title: "Text",
+    component: TextView,
+    props: {},
+  };
+
+  const asmTab = {
+    id: "asm",
+    title: "Assembly",
+    component: AssemblyView,
+    props: {},
+  };
+
+  const decompTab = {
+    id: "decomp",
+    title: "Decompilation",
+    component: DecompilationView,
+    props: {},
+  };
```

Comment:
Could make sense to put all of these inside one big object in case we need to iterate over them later for some reason.

Not worth blocking this PR on that, but figured I'd float the idea in case you like that style of managing this data better.

Reply:
I'll do this some other time.